### PR TITLE
include new videos showcasing current updates

### PR DESCRIPTION
**Pull request by kamalx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ A global chatroom for all my friend-circles brought together anonymously 🔯
 - ListViews to build scrolling views
 - Firebase Could Firestore to store and retrieve data
 
-![Finished App](https://github.com/londonappbrewery/Images/blob/master/flash_chat_flutter_demo.gif)
+<video width='290'><source src='https://github.com/kamalx/images/blob/main/flash_chat_1.webm?raw=true' type='video/webm'></video>
+<video width='290'><source src='https://github.com/kamalx/images/blob/main/flash_chat_2.webm?raw=true' type='video/webm'></video>
```
Comment (kamalx) [inline README.md]: <video> tags don't seem to work in gh md. 👎🏽

File: README.md [hunk 2]
```diff
@@ -26,8 +26,6 @@ A global chatroom for all my friend-circles brought together anonymously 🔯
 - ListViews to build scrolling views
 - Firebase Could Firestore to store and retrieve data
 
-<video width='290'><source src='https://github.com/kamalx/images/blob/main/flash_chat_1.webm?raw=true' type='video/webm'></video>
-<video width='290'><source src='https://github.com/kamalx/images/blob/main/flash_chat_2.webm?raw=true' type='video/webm'></video>
-
+<video width='290'><source src='https://mulberries.xyz/appvids/flash_chat.webm' type='video/webm'></video>
```
Comment (kamalx) [inline README.md]: still no go. it just doesn't seem to support the video tag in html at all.

**Review by ashishbhate — CHANGES_REQUESTED** [review]
use gif :joy:

**Comment by kamalx** [comment]
> use gif

this is not right. webm is < 2mb, degraded dithered 128 color gif is 14 mb. :/ 
they should provide support for webm in README files.

File: README.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ A global chatroom for all my friend-circles brought together anonymously 🔯
 - ListViews to build scrolling views
 - Firebase Could Firestore to store and retrieve data
 
-![Finished App](https://github.com/londonappbrewery/Images/blob/master/flash_chat_flutter_demo.gif)
-
+![Login](https://github.com/kamalx/images/blob/main/FlashChat1.gif)
+![Chat Screen](https://github.com/kamalx/images/blob/main/FlashChat2.gif)
```
Comment (kamalx) [inline README.md:30]: these work. आपका बँडविद्थ है. हमको क्या? हम सड़ा हुआ गिफ use कर लेंगे
please squash commits.

**Pull request merged**